### ISSUE TITLE
Initiating download from context menu crashes process

### ISSUE THREAD

**Issue by Ygrex** [aidanholm/luakit#86]
What I do:
1. LC_ALL=C ./luakit -U https://luakit.github.io/luakit
2. Find "Frequently Asked Questions" on the page, click right mouse button on it, context menu appears
3. Click on "Download Linked File"

What I see:

application halts with the following output:

```
**
ERROR:./widgets/webview/downloads.c:39:download_start_cb: assertion failed: (w)
zsh: abort      LC_ALL=C ./luakit -U https://luakit.github.io/luakit
```

What I expect:

application keeps running

**Comment by aidanholm** [comment]
Root cause: no web view associated with the download, which is unexpected behavior on WebKit's part. Probably need to move `download-start` signal to a central Lua object;`clib.luakit` is the obvious choice.

**Comment by aidanholm** [comment]
Fixed at 766a21e
